From ffc8bf8bf9afa1a8083593b76730aa9f045cc7ec Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Thu, 14 Mar 2019 15:16:36 +0100
Subject: [PATCH] imx8m_var_dart.h: Set ads7846 chipselect before imx_spi
 probes the driver

GPIO5_IO9 interferes with ECSPI1_MISO if first is not set.
Hogging it in the dtb will cause imx_spi driver to fail,
doing it after is not possible as it is already taken by driver.

Set gpio pin in u-boot, before everything else.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 include/configs/imx8m_var_dart.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/imx8m_var_dart.h b/include/configs/imx8m_var_dart.h
index c7fe6ff..e24bfc8 100644
--- a/include/configs/imx8m_var_dart.h
+++ b/include/configs/imx8m_var_dart.h
@@ -185,6 +185,7 @@
 	   "setenv mmcdev ${resin_dev_index};" \
 	   "setenv mmcbootpart ${resin_boot_part};" \
 	   "if test ${mmcdev} = 0; then " \
+		"gpio set 137; " \
 		"setenv fdt_file fsl-imx8mq-var-dart-emmc-wifi-hdmi.dtb; " \
 	   "else " \
 	   	"setenv fdt_file fsl-imx8mq-var-dart-sd-emmc-hdmi.dtb; " \
-- 
2.7.4

